README.rst

```diff
@@ -50,31 +49,30 @@ Also included are SQL drivers:
 |Hibernate5        |Java 8 or 11         |
 +------------------+---------------------+
 
-Of course, these clients require a NuoDB database to be running.  If you don't
-have NuoDB_ available yet, you can use the `NuoDB Community Edition`_.
+The use of the clients require a running NuoDB database.  If you don't have a NuoDB_ database running in your environment yet,
+you can use the NuoDB `Community Edition`_ page to guide you through the steps to deploy a NuoDB database.
 
 Installation
 ------------
 
-Unpack the tar file, or unzip the zip file.  No other installation is
-required.
+Unpack the tar file, or unzip the zip file which will create a *nuodb-client-<ver>* directory that contains the NuoDB Client
+Package files.
 
-To use the drivers you may need to configure applications with the appropriate
-paths.
+To use the drivers you may need to configure your user applications with the appropriate path settings to locate your 
+NuoDB Client package install directory at runtime.
 
 Resources
 ---------
 
-NuoDB Documentation: https://doc.nuodb.com/Latest/Default.htm
+NuoDB Documentation_
 
 License
 -------
 
-NuoDB Client is licensed under the `BSD 3-Clause License <https://github.com/nuodb/nuodb-client/blob/master/LICENSE>`_.
+NuoDB Client is licensed under the `BSD 3-Clause License <https://github.com/nuodb/nuodb-client/blob/master/LICENSE>`_
```

Comment:
It might be worthwhile to make more of a distinction here between the code in this GitHub repository, which is used to create a client package and is licensed via BSD, and the licensing of the actual client software that is downloaded by this code, which have their own licenses which should be investigated.